### PR TITLE
Feature/unit test report

### PR DESCRIPTION
**Pull request by tej-rapolu**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,12 +1,63 @@
 <?xml version="1.0" encoding="UTF-8"?>
+
+
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-
+<!-- cljkasdnckljansdc -->
```
Comment (Sathyvs) [inline pom.xml]: Is this comment valid ?

File: pom.xml [hunk 2]
```diff
@@ -1,12 +1,63 @@
...
     <groupId>com.schoolman</groupId>
     <artifactId>skyline</artifactId>
     <version>1.0.0.0-SNAPSHOT</version>
+    <packaging>war</packaging>
 
-    
+    <name>skyline</name>
+    <url>http://maven.apache.org</url>
+    <properties>
+        <spring.version>4.3.8.RELEASE</spring.version>
```
Comment (Sathyvs) [inline pom.xml]: Is this being used anywhere in the pom ?

File: pom.xml [hunk 3]
```diff
@@ -1,12 +1,63 @@
...
+                <artifactId>maven-surefire-report-plugin</artifactId>
+                <version>2.22.0</version>
+            </plugin>
+        </plugins>
+    </reporting>
+    <dependencies>
+        <dependency>
+            <groupId>jstl</groupId>
+            <artifactId>jstl</artifactId>
+            <version>1.2</version>
```
Comment (Sathyvs) [inline pom.xml]: is jstl version required for this project ?

File: pom.xml (hunk 1 above)
Comment (tej-rapolu) [reply pom.xml]: Its not a valid comment, I learned how to write a comment in IntelliJ

File: pom.xml (hunk 2 above)
Comment (tej-rapolu) [reply pom.xml]: No I used it only once

File: pom.xml (hunk 3 above)
Comment (tej-rapolu) [reply pom.xml]: I think it is required and I used the latest version

File: pom.xml (hunk 1 above)
Comment (Sathyvs) [reply pom.xml]: Okay, now that you learnt, can u please remove it from the source code ?

File: pom.xml (hunk 2 above)
Comment (Sathyvs) [reply pom.xml]: Where did you use it ? i dont see it in the pom anywhere ? This project doesnt even seem to have spring dependency

File: pom.xml (hunk 3 above)
Comment (Sathyvs) [reply pom.xml]: Can you double check and confirm it ? if this is not required pls remove it

File: pom.xml (hunk 1 above)
Comment (tej-rapolu) [reply pom.xml]: Yeah I removed it

File: pom.xml (hunk 2 above)
Comment (tej-rapolu) [reply pom.xml]: I deleted it and checked. The changes are working fine
```suggestion
        <spring.version>4.3.8.RELEASE</spring.version>
```

File: pom.xml (hunk 3 above)
Comment (tej-rapolu) [reply pom.xml]: Yeah I removed it and everything's working fine

**Pull request merged**